### ISSUE TITLE
Defund Blacksburg, VA Police

### ISSUE THREAD

**Issue by womanatee** [defund12/defund12.org#1423]
To: lhagersmith@blacksburg.gov, msutphin@blacksburg.gov, jbush@blacksburg.gov, lcolliver@blacksburg.gov, jford@blacksburg.gov, smattingly@blacksburg.gov, sanderson@blacksburg.gov, towncouncil@blacksburg.gov  

Dear Mayor Hager-Smith, Vice Mayor Michael Sutphin, and Blacksburg Town Council Members,

My name is [YOUR NAME] and I am a resident of Blacksburg. I am emailing in support of the nationwide initiative to defund police departments in favor of supporting community based public safety measures. The Adopted Budget in Brief for FY 2019 states that Blacksburg allocated $8,520,000 to our Police department, which is approximately 25% of the $34.3M General Fund Expenses and the largest expenditure by far. The Blacksburg Town Council should adopt a budget that prioritizes community well-being, and redirects funding away from the police. Additionally, given the likely budget shortfalls resulting from the pandemic, now is an urgent time to re-evaluate the priorities of our town. 

The Town Council should begin meaningfully re-allocating Blacksburg Police Department funds to programs proven to more effectively promote a safe and equitable community. We need funding for community-based mental health services, substance abuse treatment services, affordable housing programs, childcare, public health, sustainability, and capital improvement projects. Additionally, police officers should not be the first resource for every crisis. I demand a budget that reflects the needs and values of Blacksburg residents. We must take a hard look at the way the current system fails to serve-and in fact actively harms-our community, and come together to reimagine the role of police in our town.

I am urging you to keep these things in mind for the budget for the 2020-2021 fiscal year and to invest in the people, not the police.

Thank you, [YOUR NAME] [YOUR ADDRESS] [YOUR EMAIL] [YOUR PHONE NUMBER]

**Comment by mahrer** [comment]
Thank you for submitting your city to defund12.org! Your request is now in the queue for review and publishing. We’ll reply here if we have any questions.

**Comment by brickaney** [comment]
I am reviewing

**Comment by wendywx** [comment]
Thanks @brickaney for the edits! I updated the statistics as per the FY 2020-21 document. Further edits in bold:

---------------------------------------------------

To: lhagersmith@blacksburg.gov, msutphin@blacksburg.gov, jbush@blacksburg.gov, lcolliver@blacksburg.gov, jford@blacksburg.gov, smattingly@blacksburg.gov, sanderson@blacksburg.gov, towncouncil@blacksburg.gov

Dear Mayor Hager-Smith, Vice Mayor Michael Sutphin, and Blacksburg Town Council Members,

My name is [YOUR NAME] and I am a resident of Blacksburg. I am emailing in support of the nationwide initiative to defund police departments in favor of supporting community based public safety measures. The **proposed** budget for **FY 2020-21** states that Blacksburg is allocating $**9,3**M to the Police department, which is approximately 2**3**% of the $**39.9**M General Fund Expenses and the largest expenditure by far, **more than the proposed budgets of Housing & Neighborhood Services, Community Relations Office, Parks & Recreation, and Sustainability combined** [1]. The Blacksburg Town Council should adopt a budget that prioritizes community well-being, and redirects funding away from the police. Additionally, given the likely budget shortfalls resulting from the pandemic, now is an urgent time to re-evaluate the priorities of our town.

**Even though the FY 2020-21 budget has been passed as a placeholder, I urge the Town Council to amend this budget and** begin meaningfully re-allocating Blacksburg Police Department funds to programs proven to more effectively promote a safe and equitable community. Blacksburg needs funding for community-based mental health services, substance abuse treatment services, affordable housing programs, childcare, public health, sustainability, and capital improvement projects. Additionally, police officers should not be the first resource for every crisis.

I demand a budget that reflects the needs and values of Blacksburg residents. This council must take a hard look at the way the current system fails to serve-and in fact actively harms-our community, and come together to reimagine the role of police in our town.

I am urging you to keep these things in mind for the budget for the 2020-2021 fiscal year and to invest in the people, not the police.

Thank you,
[YOUR NAME]
[YOUR ADDRESS]
[YOUR EMAIL]
[YOUR PHONE NUMBER]

[1] **https://www.blacksburg.gov/home/showdocument?id=9284**

----------------------------------------

According to https://www.roanoke.com/news/nrv/blacksburg-council-approves-budget-but-town-expects-adjustments-later/article_0a490594-ee26-5969-ae5f-592a4c18008b.html the Blacksburg budget for FY 2020-21 was already approved by Blacksburg Town Council, so I also edited the wording to reflect that.

Marking this as ready for PR.
